lms/services/jstor.py

```diff
@@ -22,8 +24,45 @@ class JSTORMetadataSchema(RequestsResponseSchema):
     title = fields.Str()
     subtitle = fields.Str()
 
-    # List of titles of works that this item is a review of.
     reviewed_works = fields.List(fields.Str())
+    """List of titles of works that this item is a review of."""
+
+    has_pdf = fields.Boolean(required=True)
+    """
+    Does this item have a PDF?
+
+    This can be false if the item is a collection (eg. of book chapters or
+    journal articles).
+    """
+
+    requestor_access_level = fields.Str(required=True)
+    """
+    Does the current institution have access to the PDF?
+
+    The "current" institution is the one identified by the site code specified
+    in the Authorization header of the request.
+
+    This will be "full_access" if the institution has access to the PDF,
+    or another value (eg. "preview_access") otherwise.
+    """
+
+
+class ContentStatus(Enum):
+    """Indicates whether an item has content available for use with Hypothesis."""
+
+    AVAILABLE = auto()
+    """Content is available for the item."""
+
+    NO_CONTENT = auto()
+    """Item does not have associated content (eg. a PDF)."""
+
+    NO_ACCESS = auto()
+    """This item has content, but the current institution does not have access to it."""
```

Comment:
I think this Enum might be causing us some trouble we might be able to avoid in a couple of ways.

One of those ways is to create a string based enum, as described here: https://docs.python.org/3/library/enum.html#others

This would look like this:

```python
class ContentStatus(str, Enum):
    AVAILABLE = "available"
    NO_CONTENT = "no_content"
    NO_ACCESS = "no_access"
```

This means we would not need to do anything special in the view to have the value stringify correctly when rendered to JSON, which is nice. This does have some other effects too:

 * You can now compare directly for equality with the string value
 * `str()` and `f"{thing}"` type strings behave differently and potentially confusingly
 
If you look at point one above there, it ends up stripping out one of the main reasons you might want an Enum in the first place. Which might be pointing the fact that maybe we don't, and actually we get most of the benefit we want by having defined constants without the `Enum` wrapper:

```python
class ContentStatus:
    AVAILABLE = "available"
    NO_CONTENT = "no_content"
    NO_ACCESS = "no_access"
```

However there are still a few nice benefits, like being able to parse with the `Enum` object and iterate them. So either way.

This _might_ all lead you to think maybe we could just return the correct strings directly in the function? I think that's viable, but I prefer your approach as it means not duplicating the values in the tests.

### About `auto()`

In general we don't use `auto()` as I think the value has become short hand for the formatted version of the Enum. The auto docs specifically say to use it when you don't care about the value and if you assume it's the formatted version we do. This also hints to the reader what the reader what the formatted version is (if it's auto you're guessing) and also lets the two be different.

This isn't handed down by the gods or anything, but it's what we do elsewhere, and the natural stringification above suggests this is what the `Enum` writers might have in mind too.